XenAdminTests/CommandTests/MigrateVirtualDiskCommandTest.cs

```diff
@@ -50,7 +50,7 @@ public class MigrateVirtualDiskCommandTest : DatabaseTester_TestFixture
         [Test]
         public void CheckContextMenuText()
         {
-            MigrateVirtualDiskCommand cmd = new MigrateVirtualDiskCommand(mw, null as VDI);
+            var cmd = new MigrateVirtualDiskCommand(mw, null);

```

Comment:
I believe that this change causes a this test to fail with ArgumentNullException, as the second parameter in the constructor (Selection) cannot be null. Same for CommandCannotExecuteNullVdi.

The ContextMenu_VDI test is also failing, but probably for a different reason.